pkg/pipeline/write/write_loki_test.go

```diff
@@ -146,13 +140,18 @@ func TestTimestampScale(t *testing.T) {
 		t.Run(fmt.Sprintf("unit %v", testCase.unit), func(t *testing.T) {
 			yamlConf := fmt.Sprintf(`log-level: debug
 pipeline:
-  write:
-    type: loki
-    loki:
-      timestampScale: %s
+  - name: write1
+parameters:
+  - name: write1
+    write:
+      type: loki
+      loki:
+        timestampScale: %s
 `, testCase.unit)
-			config.Opt.PipeLine.Write.Loki = initFromString(t, []byte(yamlConf))
-			loki, err := NewWriteLoki()
+			v := test.InitConfig(t, string(yamlConf))
```

Comment:
Redundant type conversion
```suggestion
			v := test.InitConfig(t, yamlConf)
```